src/main/kotlin/app/revanced/patches/youtube/layout/theme/patch/ThemePatch.kt

```diff
@@ -45,9 +43,68 @@ class ThemePatch : ResourcePatch {
         }
 
         // copies the resource file to change the splash screen color
-        context.copyResources("theme",
-            ResourceUtils.ResourceGroup("values-night-v31", "styles.xml")
-        )
+        context.xmlEditor["res/values/attrs.xml"].use { editor ->
+            with(editor.file) {
+                val resourcesNode = getElementsByTagName("resources").item(0) as Element
+
+                val newElement: Element = createElement("attr")
+                newElement.setAttribute("format", "reference")
+                newElement.setAttribute("name", "splashScreenColor")
+
+                resourcesNode.appendChild(newElement)
+            }
+        }
+        context.xmlEditor["res/values/styles.xml"].use { editor ->
+            with(editor.file) {
+                val resourcesNode = getElementsByTagName("resources").item(0) as Element
+
+                for (i in 0 until resourcesNode.childNodes.length) {
+                    val node = resourcesNode.childNodes.item(i) as? Element ?: continue
+
+                    val newElement: Element = createElement("item")
+                    newElement.setAttribute("name", "splashScreenColor")
+
+                    when (node.getAttribute("name")) {
+                        "Base.Theme.YouTube.Launcher.Dark" -> {
+                            newElement.appendChild(createTextNode(darkThemeBackgroundColor))
+
+                            node.appendChild(newElement)
+                        }
+                        "Base.Theme.YouTube.Launcher.Light" -> {
+                            newElement.appendChild(createTextNode(lightThemeBackgroundColor));
+
+                            node.appendChild(newElement)
+                        }
+                    }
+                }
+            }
+        }
+        context.xmlEditor["res/values-v31/styles.xml"].use { editor ->
```

Comment:
Constants should be used for the resource files.

Reply:
With a list of names?

Reply:
As an example `values-v31` is not self-explanatory and contributes against readability. A constant for it or the entire resource file path `res/values-v31/styles.xml` should be used. This should be done with the other non self-explanatory strings found in this patch.

Reply:
Well so... A single string field that contains the path, with a dedicated comment to explain to what does this file correspond to.

Reply:
A comment is not necessary, the constant field name can explain it.

Reply:
In this case I can't do anything else, because I have no way to access these files and decompile them to find the folder names.

Reply:
After thinking about it for a while, I thought it would be better for the readability of the code to create a list of "paths" with comments alongside